bin/start_backend.sh

```diff
@@ -185,6 +185,21 @@ if [[ $(ulimit -n) -lt 60000 ]]; then
 fi
 
 START_BE_CMD="${NUMA_CMD} ${STARROCKS_HOME}/lib/starrocks_be"
+
+# Add be cgroup hard limit.
+if [ -f ${STARROCKS_HOME}/conf/cgroup.conf ]; then
+    source ${STARROCKS_HOME}/conf/cgroup.conf
```

Comment:
tricky to source the cgroup.conf directly.  the `.conf` extension indicate it is a plaintext conf file, may not have the correct format for shell source its env.

follow `export_env_from_conf $STARROCKS_HOME/conf/be.conf` to if you need to load env from a plaintext conf file rather than a shell script.